src/hclib-deque.c

```diff
@@ -39,32 +39,38 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hclib-internal.h"
 #include "hclib-atomics.h"
 
-void deque_init(deque_t *deq, void *init_value) {
+#if 0 // UNUSED
```

Comment:
Okay, then I'd vote for now that we delete this block and the other block I commented on, and then I'll click the merge button.